FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/JobManager.cs

```diff
@@ -120,6 +122,10 @@ private async Task<Job> CreateNewJobAsync(CancellationToken cancellationToken =
             {
                 resourceTypes = _fhirSpecificationProvider.GetAllResourceTypes();
             }
+            else
+            {
+                resourceTypes = Enumerable.Repeat("Observation", 1);
```

Comment:
What does this mean by adding Observation type here?

Reply:
Oops, this is a piece of test code. Removed